react/index.js

```diff
@@ -85,27 +85,34 @@ SearchResultQueryLoader.getSchema = props => {
     type: 'object',
     properties: {
       ...querySchema,
-      hiddenFacets: {
-        title: 'editor.search-result.hiddenFacets',
+      mobileLayout: {
+        title: 'Mobile Layout Switcher',
```

Comment:
Ids should be used in order to support internationalization

Reply:
Fixed.